Automattic/jetpack
# Social: Disable retry in share status when resharing is not supported

Resharing is not always available, thus we need to disable retry accordingly in share status modal.


## Proposed changes:

* Disable retry in share status when resharing is not supported

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply this PR to your sandbox
* Add this to `0-sandbx.php`
```php
define( 'JETPACK_SOCIAL_HAS_SHARE_STATUS', true );
```
* Point your Simple site with a free plan and the public API to your sandbox
* Draft a post ready for publishing to Facebook
* Remove "Jetpack" from [here](https://www.facebook.com/settings/?tab=business_tools) to break Facebook connections
* Now quickly publish the post
* Open share status modal for the psot
* Confirm that retry is disabled with the message saying, `"To re-share a post, you need to upgrade to a paid plan."`

<img width="376" alt="Screenshot 2024-09-26 at 5 24 15 PM" src="https://github.com/user-attachments/assets/5d2028f8-be24-4a41-87e4-68dc948be975">
